### PR TITLE
Translated new strings to Norwegian Bokmål

### PR DESCRIPTION
**Pull request by peterjohansen**

### CODE REVIEW

File: translations/tiled_nb.ts [hunk 1]
```diff
@@ -217,12 +217,12 @@
     <message>
         <location filename="../src/plugins/defold/defoldplugin.cpp" line="+59"/>
         <source>Defold files (*.tilemap)</source>
-        <translation type="unfinished"></translation>
+        <translation>Brett ut filer (*.tilemap)</translation>
```
Comment (bjorn) [inline translations/tiled_nb.ts]: Please don't translate "Defold" since it's the name of a game engine. :)

File: translations/tiled_nb.ts [hunk 2]
```diff
@@ -4717,9 +4717,9 @@ Vennligst velgt et spesifikt format.</translation>
     <message numerus="yes">
         <location filename="../src/tiled/flipmapobjects.cpp" line="+39"/>
         <source>Flip %n Object(s)</source>
-        <translation type="unfinished">
-            <numerusform></numerusform>
-            <numerusform></numerusform>
+        <translation>
+            <numerusform>Vend %n Objekt</numerusform>
```
Comment (bjorn) [inline translations/tiled_nb.ts]: Since this is the singular form, you may want to leave out the %n.

**Comment by bjorn** [comment]
Thanks for the quick update!

File: translations/tiled_nb.ts (hunk 1 above)
Comment (peterjohansen) [inline translations/tiled_nb.ts]: Ops. No wonder I couldn't figure out what defolding a map was supposed to do :P

Comment (bjorn) [inline translations/tiled_nb.ts]: Haha. :-)

**Pull request merged**